src/helpers/client/safari-client.ts

```diff
@@ -227,6 +228,15 @@ export class SafariClient implements Client {
     });
   }
 
+  /**
+   * Listens for 'isAppAlive' events.
+   */
+  private listenClientStatusEvents() {
+    document.addEventListener('isAppAlive', (event: any) => {
+      asperaSdk.activityTracking.handleClientEvents(event.detail.running);
```

Comment:
Can we add some checks before trying to access `event.detail` and `event.detail.running`?

Reply:
Sure!